doc/contributor/cexts.md

```diff
@@ -1,5 +1,10 @@
 # TruffleRuby, Sulong and C extensions
 
+## Requirements
+
+* `llvm` including the `opt` binary
+* OpenSSL 1.0.2 including the development headers
```

Comment:
Could link to https://github.com/graalvm/truffleruby/blob/master/doc/user/installing-libssl.md